evap/results/templates/results_index_course.html

```diff
@@ -22,7 +22,7 @@
                 <span data-toggle="tooltip" data-placement="top" class="fas fa-chart-bar icon-yellow" title="{% trans 'Results not yet published' %}"></span>
             {% endif %}
         {% endif %}
-        <span class="course-name">{{ course.name }}</span>
+        <span class="course-name">{{ course.name }}{% if course.is_single_result %} ({{ course.vote_start_datetime|date:"Y-m-d" }}){% endif %}</span>
```

Comment:
you should be able to leave out the parameter to `date`, the default is `DATE_FORMAT`, which is exactly what you wrote there.

Reply:
leaving that out returns `yyyy-MM-dd HH:mm`

Reply:
that is weird. https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#date says `When used without a format string, the DATE_FORMAT format specifier is used`, and from a quick glance at the code, it does just that. could you investigate?

Reply:
it's not a date, but a datetime ;)

Reply:
yes, but the `date` filter does not care as far as i can see.